securesystemslib/signer/_key.py

```diff
@@ -341,9 +341,18 @@ def _verify_ed25519_fallback(self, signature: bytes, data: bytes) -> None:
 
     def _verify(self, signature: bytes, data: bytes) -> None:
```

Comment:
The function has become a bit unwieldy. But I did resist the temptation to refactor anything, mostly because want this PR to be easy to review ... and also because I was unsure how to make it better. :)

One idea I had was to factor out the long deserialization if/else switch to something like a `_to_crypto` method: 504ffaea84ff6a6062de93cc9d277adac9cffe73

...

I'll think about this some more.